napari/utils/misc.py

```diff
@@ -121,66 +120,6 @@ def formatdoc(obj):
         del frame
 
 
-class StringEnumMeta(EnumMeta):
```

Comment:
I'd be tempted to move these into a new file that is not the actual file with the constants either, might keep things most clear/ readable

Reply:
that's fine, they can also go back to utils.misc where they came from